### PR TITLE
feat: Portals store integration

### PR DESCRIPTION
**Pull request by muhsin-k**
Fixes https://github.com/chatwoot/product/issues/488

**Screencast**

https://www.loom.com/share/6e2c5aeafb51464eb1b4bdfea3e44458

### CODE REVIEW

File: app/javascript/shared/constants/locales.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  { code: 'cv', name: 'Chuvash', nativeName: 'чӑваш чӗлхи' },
+  { code: 'kw', name: 'Cornish', nativeName: 'Kernewek' },
+  { code: 'co', name: 'Corsican', nativeName: 'corsu, lingua corsa' },
+  { code: 'cr', name: 'Cree', nativeName: 'ᓀᐦᐃᔭᐍᐏᐣ' },
+  { code: 'hr', name: 'Croatian', nativeName: 'hrvatski' },
+  { code: 'cs', name: 'Czech', nativeName: 'česky, čeština' },
+  { code: 'da', name: 'Danish', nativeName: 'dansk' },
+  { code: 'dv', name: 'Divehi; Dhivehi; Maldivian;', nativeName: 'ދިވެހި' },
+  { code: 'nl', name: 'Dutch', nativeName: 'Nederlands, Vlaams' },
+  { code: 'en', name: 'English', nativeName: 'English' },
```
Comment (pranavrajs) [inline app/javascript/shared/constants/locales.js:40]: Can we have variants as well? (en-us, en-GB)

Comment (muhsin-k) [reply app/javascript/shared/constants/locales.js:40]: Sure

Comment (muhsin-k) [reply app/javascript/shared/constants/locales.js:40]: @pranavrajs I am planning to use all locales from this list https://github.com/umpirsky/locale-list/blob/master/data/en/locales.json

**Review by iamsivin — COMMENTED** [review]
LGTM 👍🏻

**Review by nithindavid — APPROVED** [review]

**Review by iamsivin — APPROVED** [review]

**Review by iamsivin — APPROVED** [review]

**Pull request merged**